geoips/sector_utils/tc_tracks.py

```diff
@@ -140,6 +144,41 @@ def get_tc_long_description(area_id, fields):
     return long_description
 
 
+def set_volc_area_def(fields, storm_name, sector_spec_template):
```

Comment:
This is currently what was required to get a non tc dynamic sector working.  We'll generalize this process, and this update identifies required updates